### PR TITLE
explosions now only hard knockdown for 1-4 seconds instead of 4-16+, instead doing stamina damage. on people affected.

### PR DESCRIPTION
**Pull request by silicons**
## Why It's Good For The Game

hard knockdown under stamina combat is bad and jarring, time to change that.

## Changelog
:cl: silicons
balance: getting hit by an explosion will now barely hard knockdown, but will leave you somewhat winded.
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
🧂

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -336,7 +336,8 @@
 			if (!istype(ears, /obj/item/clothing/ears/earmuffs))
 				adjustEarDamage(30, 120)
 			Unconscious(20)							//short amount of time for follow up attacks against elusive enemies like wizards
-			Knockdown(200 - (bomb_armor * 1.6)) 	//between ~4 and ~20 seconds of knockdown depending on bomb armor
+			Knockdown((200 - (bomb_armor * 1.6)) / 4) 	//between ~1 and ~5 seconds of knockdown depending on bomb armor
```
Comment (Linzolle) [inline code/modules/mob/living/carbon/human/human_defense.dm:339]:
```suggestion
			Knockdown(50 - (bomb_armor * 0.4)) 	//between ~1 and ~5 seconds of knockdown depending on bomb armor
```
technically performance improvement cuz less division

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 2]
```diff
@@ -345,7 +346,8 @@
 			damage_clothes(max(50 - bomb_armor, 0), BRUTE, "bomb")
 			if (!istype(ears, /obj/item/clothing/ears/earmuffs))
 				adjustEarDamage(15,60)
-			Knockdown(160 - (bomb_armor * 1.6))		//100 bomb armor will prevent knockdown altogether
+			Knockdown((160 - (bomb_armor * 1.6)) / 4)		//100 bomb armor will prevent knockdown altogether
```
Comment (Linzolle) [inline code/modules/mob/living/carbon/human/human_defense.dm:349]:
```suggestion
			Knockdown(40 - (bomb_armor * 0.4))		//100 bomb armor will prevent knockdown altogether
```
same thing

**Review by Ghommie — APPROVED** [review]
Lgtm.

**Comment by Ghommie** [comment]
>Forgot to commit the micro-improvement suggestions.

Sorry Lin.

**Pull request merged**